RecoTracker/LSTCore/standalone/Makefile

```diff
@@ -9,8 +9,8 @@ OBJECTS_ROCM=$(SOURCES:.cc=_rocm.o)
 OBJECTS=$(OBJECTS_CPU) $(OBJECTS_CUDA) $(OBJECTS_ROCM)
 
 CXX         = g++
-CXXFLAGS    = -g -O2 -Wall -fPIC -Wshadow -Woverloaded-virtual -Wno-unused-function -fno-var-tracking -std=c++17 -DLST_IS_CMSSW_PACKAGE
-INCLUDEFLAGS= -ISDL -I$(shell pwd) -Icode -Icode/core -I${ALPAKA_ROOT}/include -I/${BOOST_ROOT}/include $(shell rooutil-config --include) -I$(shell root-config --incdir) -I${TRACKLOOPERDIR}/../../../ -I${CMSSW_BASE}/src -I../interface/alpaka/ -I../src/alpaka/
```

Comment:
@GNiendorf Seems like while migrating your changes you changed these lines unintentionally. This should be why it's failing to compile.

Reply:
Nice, thank you for finding that.